docs/source/running-autometa.rst

```diff
@@ -2,108 +2,271 @@
+
+*params.clustering_method* : Cluster contigs using which clustering method. Choices are "dbscan" and "hdbscan" (default is "dbscan")
+
+*params.binning_starting_rank* : Which taxonomic rank to start the binning from. Choices are "superkingdom", "phylum", "class", "order", "family", "genus", "species" (default is "superkingdom")
+
+*params.classification_method* : Which clustering method to use for unclustered recruitment step. Choices are "decision_tree" and "random_forest" (default is "decision_tree")
+
+*params.completeness* :  Minimum completeness needed to keep a cluster (default is atleast 20% complete)
+
+*params.purity* : Minimum purity needed to keep a cluster (default is atleast 95% pure)
+
+*params.cov_stddev_limit* : Which clusters to keep depending on the covergae std.dev (default is 25%)
+
+*params.gc_stddev_limit* : Which clusters to keep depending on the GC% std.dev (default is 5%)
+
+Running the pipeline
+--------------------
+
+You can run autometa using nextflow_ in multiple ways. You can install nexflow using conda by doing ``conda install -c conda-forge nextflow`` or you can also install from source using the instructions mentioned in their `documentation <https://www.nextflow.io/docs/latest/getstarted.html#installation>`_
+
+.. note::
+    1. Run the following commands directly in ``bash``. Nexflow will automatically submit jobs to SLURM or any other sheduling system.
+    2. You can use `tmux <https://github.com/tmux/tmux/wiki>`_ or `screen <https://www.gnu.org/software/screen/>`_ in case you want to exit the window or disconnect from the server.
+    3. The pipeline must be launched from a node where the ``condor_submit`` command is available, that is, in a common usage scenario, the cluster head node.
+    4. The HTCondor executor for Nextflow_ does not support at this time the HTCondor ability to transfer input/output data to the corresponding job computing node. Therefore the data needs to be made accessible to the computing nodes using a shared file system directory from where the Nextflow_ workflow has to be executed (or specified via the -w option).
+
+With docker (Simplest)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Make sure that you have Docker_ and nextflow_ installed. You can run autometa pipeline using ``nextflow run KwanLab/Autometa -c parameters.config``
+
+In case you want to tweak some of the scripts and modify the pipeline you can clone the repository and then run autometa using ``main.nf``. This is completely optional and would be useful only for development purposes.
+
+.. code-block:: bash
+
+    # Clone the autometa repository
+    git clone git@github.com:KwanLab/Autometa.git
+    # Navigate to the clone repository
+    cd $HOME
+    # Run nextflow
+    nextflow run main.nf
+
+Without docker
+^^^^^^^^^^^^^^
+
+By default autometa's implementation using nextflow_ makes use of Docker_. You can diasble it in either your ``nextflow.config`` file or the ``parameters.config`` file.
+
+Editing ``nextflow.config``:
```

Comment:
Where is this nextflow.config file coming from? So far there has been no mention of where to find this